packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SeparateFramesViewer/SeparateFramesViewer.js

```diff
@@ -46,14 +89,11 @@ function SeparateFramesViewer({
             onReady={onReady}
           />
         ))}
-      </Box>
-      <Box
-        justify='center'
-        pad='xsmall'
-      >
+      </Grid>
+      <Box justify='center' pad='xsmall'>
         <ViewModeButton />
```

Comment:
The ViewModeButton is expanding horizontally in a way that changes the cursor from a an arrow to pointer. It doesn't respond to clicks, but unexpected.
![ViewModeButton](https://user-images.githubusercontent.com/12118751/234357992-c59e53b8-85ff-4ec5-9b18-24797f222f30.png)

Reply:
Good point, I'll fix this styling before merging